src/parser/ast.rs

```diff
@@ -659,6 +696,14 @@ impl Expr {
             Expr::Call(ex) => ex.func.return_type,
         }
     }
+
+    /// only Some if expr is NumberLiteral
```

Comment:
This should add a link to the variant. But you might need to confirm the output doc
```suggestion
    /// only Some if expr is [Expr::NumberLiteral]
```